lib/tests/test_git.rs

```diff
@@ -19,6 +19,7 @@ use std::{fs, iter, thread};
 
 use assert_matches::assert_matches;
 use git2::Oid;
+use insta::{assert_debug_snapshot, assert_snapshot};
```

Comment:
nit: we usually spell `insta::assert_*()` at call sites (I don't know why, though.)

Reply:
FWIW, I wouldn't mind a PR that consistently imports the symbols.

Reply:
I'm OK with either convention. Explicit `insta::` might remind new people this is related to `cargo insta`.

Reply:
I don't mind either, but I think most of the insta assertions in `test_git.rs` can be removed.